dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java

```diff
@@ -464,62 +464,80 @@ private void handleDropEvent(final DragEvent event, final TreeTableRow<Ds3TreeTa
      * @param row   row
      */
     private void setBehaviorOnMouseClick(final MouseEvent event, final TreeTableRow<Ds3TreeTableValue> row) {
-        if (event.isControlDown() || event.isShiftDown() || event.isShortcutDown()) {
-            if (!rowNameList.contains(row.getTreeItem().getValue().getName())) {
-                rowNameList.add(row.getTreeItem().getValue().getName());
-                ds3TreeTable.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
-                ds3TreeTable.getSelectionModel().select(row.getIndex());
+        if (row != null && row.getTreeItem() != null && row.getTreeItem().getValue() != null) {
```

Comment:
lets move this out to its own condition that short circuit returns instead of nesting